### ISSUE TITLE
Most of the metrics are to 0.0

### ISSUE THREAD

**Issue by damien-reyre** [memgraph/prometheus-exporter#3]
Hello,

Thanks for adding the memgraph-exporter. It can be very useful.

I just noticed, that practically all my metrics are zero except for a few.

memgraph is installed in docker-compose. `1.7-memgraph-2.8-dev`

I passed the two commands for adding the metrics:
`--metrics-address`
`--metrics-port`

For the moment, I just use the python script.

The metrics **available**:
![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/756b1874-6623-445e-9528-b4f04a58ca61)

The metrics are **zero**:
![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/deb74451-cfe3-47c1-b099-25170b047750)

Until the **end**, all metrics are **zero**.

Did I forget something for having all the metrics?

Thanks

**Comment by damien-reyre** [comment]
All the metrics from memgraph itself are not available. The first metrics available are from the host (I suppose).

I connected to the container in bash and I tried to `curl localhost:9091`
![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/a8ee7626-1f69-47d9-b200-78c3d172e23a)

That the reason I can't see the metrics, but why?

If I don't add the settings for the containers `--metrics-address` and `--metrics-port`. I get a **connection refused**.

**Comment by Josipmrden** [comment]
Hello @damien-reyre , there are a couple of things which might be wrong with the setup. The workflow of enabling exposed metrics to the outside is as follows:
1. Add the enterprise license to memgraph (this is done at runtime)
2. Restart Memgraph
3. try to curl via postman

The documentation at https://memgraph.com/docs/memgraph/reference-guide/exposing-system-metrics provides you with the steps of how to enable exposing metrics. 

After doing these steps, I was successfully getting the JSON response of the metrics from Postman.

**Comment by Josipmrden** [comment]
@damien-reyre  An additional question, which metrics do you not expect to be zero, but were zero when you got the metrics response?

**Comment by damien-reyre** [comment]
Hello @Josipmrden,

My mistake was about the enterprise license. I didn't set up. Thanks.

**Comment by damien-reyre** [comment]
Hello @Josipmrden,

I got an enterprise license.

![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/544b241c-0b45-4d7b-ba11-0eb2e8efac88)

I configured the memgraph with docker compose:

![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/39b1bceb-ddac-43a9-b1cd-07c5dac7e288)

In the second step, you said:
![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/042303dd-58d3-4ddf-9a15-5c9a3fe8c708)

` curl http://172.18.0.1:9091`
`curl: (7) Failed to connect to 172.18.0.1 port 9091: Connection refused`

`curl http://172.18.0.1:9091/metrics`
`curl: (7) Failed to connect to 172.18.0.1 port 9091: Connection refused`

There is no firewall in this dev environment.

Nothing in the logs from memgraph.

I think I missed something to configure.

**Comment by damien-reyre** [comment]
Do you have any idea? @Josipmrden

**Comment by Josipmrden** [comment]
hello @damien-reyre , have you restarted Memgraph after configuring the enterprise licence?

**Comment by damien-reyre** [comment]
@Josipmrden Yes. It seems I can't get the metrics from the memgraph container.

**Comment by damien-reyre** [comment]
Finally, the parameters in commands seems not working in my case from docker-compose.yaml.

So I passed the memgraph.conf in volume and set `--metrics-address=0.0.0.0` and `--metrics-port=9091`.

This time with curl, the port is well open, but I have nothing, no metrics, no JSON.

![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/2535512d-40e1-4273-bfe9-d18a3e5cfcb1)

When I'm looking to the logs:

![image](https://github.com/memgraph/prometheus-exporter/assets/117079206/89e4280e-096d-4f64-8bc0-62f63dc92d3a)

That's possible to get this symptom with an invalid license or my organization's name is wrong?

@Josipmrden

**Comment by as51340** [comment]
Should be fixed with the latest version
